book/visual-effects.md

```diff
@@ -484,14 +485,16 @@ class DrawRect:
         canvas.drawRect(self.rect.makeOffset(0, -scroll), paint)
 ```
 
-Here the `rect` field needs to become a Skia `Rect` object, which you can
-construct using `MakeLTRB` (for "make left-top-right-bottom") or `MakeXYWH`
-(for "make *x*-*y*-width-height"). Get rid of the old `Rect` class that was
-introduced in [Chapter 7](chrome.md) in favor of `skia.Rect`. Everywhere
-that a `Rect` was constructed, instead put `skia.Rect.MakeLTRB`, and
-everywhere that the sides of the rectangle (e.g. `left`) where checked,
-replace them with the corresponding function on a Skia `Rect` (e.g. `left()`).
-Also replace calls to `containsPoint` with Skia's `contains`.
+Here the `rect` field needs to become a Skia `Rect` object.
+
+Get rid of the old `Rect` class that was introduced in [Chapter
+7](chrome.md) in favor of `skia.Rect`. Everywhere that a `Rect` was
+constructed, instead put `skia.Rect.MakeLTRB` (for "make
+left-top-right-bottom") or `MakeXYWH` (for "make
+*x*-*y*-width-height"). Everywhere that the sides of the
```

Comment:
```suggestion
x-y-width-height"). Everywhere that the sides of the
```

Reply:
Don't really know why, but we always italicize *x* and *y*. Probably it is ultimately the typographers who will tell us what to do here.